common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                "short_description": _("Other Instructional Material"),
                 "items": [
                     {
-                        "short_description": _("Explore the Studio Help Forum"),
+                        "short_description": _("Verify third-party material is freely available"),
                         "long_description": _(
-                            "Access the Studio Help forum from the menu that appears when you click your user name "
-                            "in the top right corner of Studio."
+                            "Ensure all required third party content is freely available to students (e.g. open source, public"
+                            "domain, cleared/paid by institution, etc.) and integrated into courseware (not linked out)"
                         ),
                         "is_checked": False,
-                        "action_url": "http://help.edge.edx.org/",
-                        "action_text": _("Visit Studio Help"),
-                        "action_external": True,
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
+                        "action_external": False,
                     },
                     {
-                        "short_description": _("Enroll in edX 101"),
-                        "long_description": _("Register for edX 101, edX's primer for course creation."),
+                        "short_description": _("Verify paid content (if applicable)"),
+                        "long_description": _(
+                            "Verify that any paid material in the course is optional (not required to achieve learning goals)"
+                            "and clearly marked as paid content."
+                        ),
                         "is_checked": False,
-                        "action_url": "https://edge.edx.org/courses/edX/edX101/How_to_Create_an_edX_Course/about",
-                        "action_text": _("Register for edX 101"),
-                        "action_external": True,
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
+                        "action_external": False,
                     },
                     {
-                        "short_description": _("Download the Studio Documentation"),
-                        "long_description": _("Download the searchable Studio reference documentation in PDF form."),
+                        "short_description": _("Clearly indicate off-platform content"),
+                        "long_description": _(
+                            "If you've linked learners to third-party tools outside of the edX course, ensure that you've"
+                            "clearly communicated that this linked content is outside of the edX platform and any terms of"
+                            "service of the third-party service apply."
```

Comment:
If your course includes a link to a third-party tool, verify that it is clear that the link will take learners to a site outside of your course and the edX platform, and that any terms of service required by that third party apply.